_posts/2023-04-03-security-device-cli-support.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Introducing Security Device support on RubyGems’ CLI
+layout: post
+author: Jenny Shen
+author_email: jenny.shen@shopify.com
+---
+
+In a world where many maintainer accounts are being targeted to distribute malicious code via our packages, it is crucial to add more measures that will help prevent these account takeovers. At the end of 2022, we [announced](https://blog.rubygems.org/2022/12/21/introducing-hardware-security-token-and-passkey-support.html) the addition of hardware security token and passkey support (aka WebAuthn) to help secure your RubyGems’ account in the browser.
+
+Starting today, you can now use your registered security device as a multi-factor method on the RubyGems CLI! This feature is available in [RubyGems 3.4.11](https://blog.rubygems.org/#2023) and above.
```

Comment:
May need to update RG version depending on which version the feature gets released in.

Reply:
Seems you already took care of this! 💪